test/e2e/windows/cpu_limits.go

```diff
@@ -82,10 +82,16 @@ var _ = SIGDescribe("[Feature:Windows] Cpu Resources [Serial]", func() {
 					found = true
 					break
 				}
-				framework.ExpectEqual(found, true, "Found pod in stats summary")
+				if !found{
+					framework.Failf("Pod with name : %v and namespace : %v not found in the stats summary %+v", p.Name, p.Namespace, allPods)
```

Comment:
I'm not a fan of ` : ` as separator between name/value pairs. Also name/namespace pairs usually just get printed as `<namespace>/<name>`.

```suggestion
					framework.Failf("Pod with name : %v and namespace : %v not found in the stats summary %+v", p.Name, p.Namespace, allPods)
```
```suggestion
					framework.Failf("Pod %s/%s not found in the stats summary %+v", p.Namespace, p.Name, allPods)
```